### PR TITLE
Article: Quick tip: Creating valid and accessible links

### PR DESCRIPTION
**Pull request by emmasax**
Closes issue https://github.com/a11yproject/a11yproject.com/issues/741: Adding a quick tip article about creating valid and accessible links

### CODE REVIEW

File: _posts/2019-02-15-creating-valid-and-accessible-links.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Ultimately, do not use the `title` attribute on `<a>` elements.
+
+## Focus state and keyboard
+
+Some developers/designers see the focus outline of links as ugly and remove them.
+People navigating using the keyboard require this focus state to keep track of where they are. Best practice is to <a href="https://a11yproject.com/posts/never-remove-css-outlines/">never remove CSS outlines</a> however there are accessible solutions to styling.
+
+The `<a>` element as standard can be activated by pressing the <kbd>Enter</kbd> key, so be mindful if you are attaching scripting not to override this functionality.
+
+## When an `<a>` is not suitable
```
Comment (scottaohara) [inline _posts/2019-02-15-creating-valid-and-accessible-links.md]: maybe change this to "Hints you should be using another element"?   Or "When you should use a `button` instead"

File: _posts/2019-02-15-creating-valid-and-accessible-links.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+> &ldquo;If you want to hide content from mobile and tablet users as well as assistive tech users and keyboard only users, use the title attribute.&rdquo; - <a href="https://developer.paciellogroup.com/blog/2010/11/using-the-html-title-attribute/
+">Using the HTML title attribute - The Paciello Group</a>
+
+Ultimately, do not use the `title` attribute on `<a>` elements.
+
+## Focus state and keyboard
+
+Some developers/designers see the focus outline of links as ugly and remove them.
+People navigating using the keyboard require this focus state to keep track of where they are. Best practice is to <a href="https://a11yproject.com/posts/never-remove-css-outlines/">never remove CSS outlines</a> however there are accessible solutions to styling.
```
[the remark this answers is not in the document]
Comment (emmasax) [reply _posts/2019-02-15-creating-valid-and-accessible-links.md]: The linked article mentions the solutions, so I'll make this clearer in the copy.

File: _posts/2019-02-15-creating-valid-and-accessible-links.md (hunk 1 above)
Comment (emmasax) [reply _posts/2019-02-15-creating-valid-and-accessible-links.md]: I went back and forth over those sorts of headings :)

**Comment by emmasax** [comment]
@scottaohara Thanks so much for the useful feedback.

**Comment by emmasax** [comment]
@scottaohara is this ok to go now?

**Comment by scottaohara** [comment]
hey @emmasax 

Sorry for letting this go, been a bit busy.  I made some very [minor edits](https://github.com/a11yproject/a11yproject.com/pull/742/commits/9a6a075641d7ba951ea3e2db1d99c13ce182eb2f) but it's all good to merge now. 

Thank you!

**Pull request merged**